view/next-project/src/pages/budgets/index.tsx

```diff
@@ -276,64 +300,65 @@ export default function BudgetList(props: Props) {
                       </tr>
                     </thead>
                     <tbody>
-                      {filteredExpenses.map((expenseView, index) => (
-                        <tr
-                          key={expenseView.expense.id}
-                          className={clsx(
-                            index !== expenses.length - 1 && 'border-b',
-                            'py-3 text-black-600',
-                          )}
-                        >
-                          <td
-                            className='py-3 text-center'
-                            onClick={() => onOpen(expenseView.expense.id || 0, expenseView)}
-                          >
-                            {expenseView.expense.name}
-                          </td>
-                          <td
-                            onClick={() => onOpen(expenseView.expense.id || 0, expenseView)}
-                            className='py-3 text-center'
-                          >
-                            {expenseView.expense.totalPrice}
-                          </td>
-                          <td
-                            onClick={() => onOpen(expenseView.expense.id || 0, expenseView)}
-                            className='py-3 text-center'
-                          >
-                            {formatDate(
-                              expenseView.expense.createdAt ? expenseView.expense.createdAt : '',
+                      {expenseViews &&
+                        expenseViews.map((expenseView, index) => (
+                          <tr
+                            key={expenseView.expense.id}
+                            className={clsx(
+                              index !== expenses.length - 1 && 'border-b',
```

Comment:
```suggestion
                              expenses && index !== expenses.length - 1 && 'border-b',
```
同様に、expensesがnullなときにエラーが出ていたためです。

Reply:
修正しました
[[fix]バグ修正](https://github.com/NUTFes/FinanSu/pull/681/commits/77feb1c9d47f11f29c03f4c567985195627126c7)